assets/scene/main.tscn

```diff
@@ -0,0 +1,40 @@
+[gd_scene load_steps=5 format=3 uid="uid://bmyhbnmo3jhkv"]
+
+[ext_resource type="Script" path="res://data/main.gd" id="1_pra24"]
+[ext_resource type="PackedScene" uid="uid://1ujfoei6r514" path="res://data/modules/tactics/camera/camera.tscn" id="2_j6uwt"]
+[ext_resource type="PackedScene" uid="uid://bky8ul2628kxw" path="res://data/modules/tactics/controls/controls.tscn" id="3_swndv"]
+
+[sub_resource type="ImageTexture" id="ImageTexture_sa2p4"]
+
+[node name="Main" type="Node"]
+script = ExtResource("1_pra24")
+
+[node name="Camera" type="Node3D" parent="."]
+
+[node name="TacticsCamera" parent="Camera" instance=ExtResource("2_j6uwt")]
+
+[node name="UI" type="Control" parent="."]
+layout_mode = 3
+anchors_preset = 0
+offset_right = 40.0
+offset_bottom = 40.0
+
+[node name="MapSelector" type="Control" parent="UI"]
+anchors_preset = 0
+offset_right = 161.0
+offset_bottom = 307.0
+
+[node name="LoadMap0" type="Button" parent="UI/MapSelector"]
+layout_mode = 0
+offset_left = 517.0
+offset_top = 302.0
+offset_right = 643.0
+offset_bottom = 362.0
+text = "Map 1"
+icon = SubResource("ImageTexture_sa2p4")
```

Comment:
nit: (quite sure i'm the only one affected by this haha) but for some reason when i switch to fullscreen this button stops working. i've been testing this on a macbook (probably that's why)

Reply:
also, the button got decentralized

Reply:
Replicated! (And fixed)

It was a Node config issue, thanks for catching it!
I'll submit both fixes in an upcoming merge :D